data/scenarios/02Tutorials/03-grab-place.yaml

```diff
@@ -0,0 +1,52 @@
+name: Grab and place
+description: |
+  Learn how to interact with the world by grabbing entities and placing them.
+entities:
+  - name: goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - Previously you learned how to make new things (like logger) from ingredients.
+      Now you will learn how to obtain as many ingredients as you need.
+    - There is a tree in the corner and a new one will slowly grow there once you grab it.
+    - Your goal is to collect 10 trees.
+    - You can speed this up by placing the tree to all available cells with the 'place'
+      command. That way you can have three times as many trees growing at once.
+win: |
+  try {
+    t <- as base {count "tree"};
+    return (t >= 10);
+  } { return false }
+solution:
+  move; move; t <- grab; turn back; move; place t; grab; move; place t; grab;
+  wait 2048; turn back; grab; move; grab; move; grab;
```

Comment:
@byorgey I was surprised how fast you can plant trees and how slowly they grow :laughing:

Reply:
This could also be a good place to point out that the player can adjust the game speed (ticks per second) by typing `<` / `>` on the world panel.

Reply:
I would prefer they thought that using `place` and planting trees is a really good idea. :deciduous_tree: 
We can tell them in a later challenge. :smile: ("Remember how you spent 5 minutes waiting for trees to grow? Well...")

But I could also make a "faster tree" entity here.

Reply:
Yeah, maybe a "fast tree" is the way to go here.